src/settings/FeeFineSettings.js

```diff
@@ -230,6 +240,24 @@ class FeeFineSettings extends React.Component {
     return this.props.mutator.owners.PUT(owner);
   }
 
+  getNotice = (noticeTypeId, noticeType) => {
+    const { templates } = this.state;
+    const defaultNotices = this.getDefaultNotices();
+    const defaultNoticeId = defaultNotices[`default${noticeType}NoticeId`];
+    const defaultMessage = <FormattedMessage id="ui-users.settings.default" />;
+    let templateName = templates.find(t => t.id === noticeTypeId) || {};
+
+    if (noticeTypeId) {
+      templateName = templateName?.name;
+    } else if (!noticeTypeId && defaultNoticeId) {
+      templateName = defaultMessage;
+    } else {
+      templateName = '-';
```

Comment:
Please use the [NoValue](https://github.com/folio-org/stripes-components/tree/master/lib/NoValue) component here to show `-` in an a11y-friendly way.

`getNotice()` cleans up the duplicate template-finding logic neatly. Nice work!